pandas/_libs/window/aggregations.pyx

```diff
@@ -183,14 +149,15 @@ cdef inline void remove_sum(float64_t val, int64_t *nobs, float64_t *sum_x) nogi
 
 
 def roll_sum_variable(ndarray[float64_t] values, ndarray[int64_t] start,
-                      ndarray[int64_t] end, int64_t minp,
-                      bint is_monotonic_bounds=True):
+                      ndarray[int64_t] end, int64_t minp):
     cdef:
         float64_t sum_x = 0
         int64_t s, e
         int64_t nobs = 0, i, j, N = len(values)
         ndarray[float64_t] output
+        bint is_monotonic_bounds
 
+    is_monotonic_bounds = np.all(np.diff(start) > 0) and np.all(np.diff(end) > 0)
```

Comment:
is this call expensive?  we might have something in _libs.algos that short-circuits this check

Reply:
Good call. Was able to use `is_monotonic` in ` _libs.algos`